capstone-app/src/app/modules/maps/map/map.component.html

```diff
@@ -0,0 +1,4 @@
+<google-map
+[center] = "location"
```

Comment:
Ideally I think all styling should be in your CSS file.

Reply:
this isn't styling this setting up the map location

Reply:
What about the height and width attributes below? I think those can go in CSS

Reply:
height and width I will move to css

Reply:
Yes, sorry, I was referring to the height and width.

Reply:
Altering the height and width of the Google Map component won't change the size of the map itself. I recommend going back to the height and width attributes.

Separately, the attributes (center) should be indented

Reply:
yes I noticed the moving height and width to css won't change the maps attributes.